source/javascripts/services/_appService.js.erb

```diff
@@ -564,6 +566,40 @@
 				return $q.when();
 			};
 
+			appService.getBetaTags = function() {
+				if (appService.betaTags) {
+					return $q.when();
+				}
+
+				if (requestService.mode === "website") {
+					return requestService.getBetaTags()
+						.then(function(betaTags) {
+							appService.betaTags = betaTags;
+						}).catch(() => {
+							appService.betaTags = null;
```

Comment:
Don't swallow the error with empty beta tag config, let it propagate so it's showing up on the UI.